packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -326,6 +334,23 @@ export const prepareLiquidityPoolKit = (zone, zcf, USDC, tools) => {
           external.publishPoolMetrics();
         },
       },
+      withdrawFeesHandler: {
+        /** @param {ZCFSeat} seat */
+        async handle(seat) {
+          const { feeSeat } = this.state;
+
+          const proposal = seat.getProposal();
+          const { want } = proposal;
```

Comment:
`proposal` isn't necessary and makes more for the reader to track.
```suggestion
          const { want } = seat.getProposal();
```
then below,
```js
zcf.atomicRearrange(harden([[feeSeat, seat, want]]));
```